bitrise.yml

```diff
@@ -35,85 +35,99 @@ workflows:
     - git-clone@4: {}
     - script@1:
         inputs:
-        - content: |-
-            sudo update-alternatives --set javac /usr/lib/jvm/java-11-openjdk-amd64/bin/javac
-            sudo update-alternatives --set java /usr/lib/jvm/java-11-openjdk-amd64/bin/java
+        - content: >-
+            sudo update-alternatives --set javac
+            /usr/lib/jvm/java-11-openjdk-amd64/bin/javac
+
+            sudo update-alternatives --set java
+            /usr/lib/jvm/java-11-openjdk-amd64/bin/java
+
             export JAVA_HOME='/usr/lib/jvm/java-11-openjdk-amd64'
-            envman add --key JAVA_HOME --value '/usr/lib/jvm/java-11-openjdk-amd64'
+
+            envman add --key JAVA_HOME --value
+            '/usr/lib/jvm/java-11-openjdk-amd64'
     - cache-pull@2: {}
     - install-missing-android-tools@2:
         inputs:
-        - gradlew_path: "$PROJECT_LOCATION/gradlew"
+        - gradlew_path: $PROJECT_LOCATION/gradlew
     - android-build@0:
         inputs:
-        - variant: "$INTEGRATOR_VARIANT"
-        - module: "$EXAMPLE_APP_MODULE"
+        - variant: $INTEGRATOR_VARIANT
+        - module: $EXAMPLE_APP_MODULE
     - android-lint@0:
         inputs:
-        - project_location: "$PROJECT_LOCATION"
-        - module: "$WIDGET_SDK_MODULE"
-        - variant: "$SDK_VARIANT"
+        - project_location: $PROJECT_LOCATION
+        - module: $WIDGET_SDK_MODULE
+        - variant: $SDK_VARIANT
         title: Run Lint for SDK
     - android-unit-test@1:
         inputs:
-        - project_location: "$PROJECT_LOCATION"
-        - module: "$WIDGET_SDK_MODULE"
-        - variant: "$SDK_VARIANT"
+        - project_location: $PROJECT_LOCATION
+        - module: $WIDGET_SDK_MODULE
+        - variant: $SDK_VARIANT
         title: Unit Test for SDK
     - android-lint@0:
         inputs:
-        - project_location: "$PROJECT_LOCATION"
-        - module: "$EXAMPLE_APP_MODULE"
-        - variant: "$INTEGRATOR_VARIANT"
+        - project_location: $PROJECT_LOCATION
+        - module: $EXAMPLE_APP_MODULE
+        - variant: $INTEGRATOR_VARIANT
         title: Run Lint for APP
     - android-unit-test@1:
         inputs:
-        - project_location: "$PROJECT_LOCATION"
-        - module: "$EXAMPLE_APP_MODULE"
-        - variant: "$INTEGRATOR_VARIANT"
+        - project_location: $PROJECT_LOCATION
+        - module: $EXAMPLE_APP_MODULE
+        - variant: $INTEGRATOR_VARIANT
         title: Unit Test for APP
     - deploy-to-bitrise-io@1:
         inputs:
-        - notify_email_list: "$BUILD_EMAILS"
+        - notify_email_list: $BUILD_EMAILS
     - cache-push@2: {}
     envs:
     - opts:
         is_expand: false
       INTEGRATOR_VARIANT: master
   increment_project_version:
```

Comment:
Maybe I missed that before but if I understand correctly this workflow is expected to be run as a part of other workflows and not as a standalone workflow. In this case, I would recommend renaming it to start with an underscore to hide this workflow in some Bitrise UI parts not to confuse people.
> The Bitrise CLI supports a small notation, called utility Workflow: a Workflow whose ID starts with an underscore character, for example, _setup. A utility Workflow can't be executed directly with a bitrise run command: they can be referenced with before_run and after_run properties.

Reply:
Added it here now https://github.com/salemove/android-sdk-widgets/pull/372/files#diff-61d1ee831cb1df242161a7d23171914a1d5d628c7659b52db957d80a9394bd68R89